recipes/element/webview.js

```diff
@@ -2,9 +2,20 @@ module.exports = Ferdium => {
   function getMessages() {
     let directCount = 0;
     const spacesBar = document.querySelector('.mx_SpaceTreeLevel');
-    for (const badge of spacesBar.querySelectorAll('.mx_NotificationBadge_count')) {
-      directCount += Ferdium.safeParseInt(badge.textContent);
+
+    for (const badge of spacesBar.querySelectorAll(
+      '.mx_SpaceItem:not(.mx_SpaceItem_narrow):first-of-type .mx_NotificationBadge_count, '
+        + '.mx_SpaceItem_narrow .mx_NotificationBadge_count'
+    )) {
+      const badgeContent = badge.textContent.toLowerCase();
+      directCount += badgeContent.endsWith('k')
+        ? Number.parseFloat(badgeContent) * 1000 + Number.parseInt(
```

Comment:
I dont understand the 2nd `Number.parseInt` that's being added after the multiplication by 1000. Can you please explain with an example?

Reply:
If the k-number integer has one digit, the closest approximation is 50: 1.2 * 1000 + 50 = 1250.

If the k-number integer has two digits, the closest approximation is 500: 12 * 1000 + 500 = 12500 (because 12500 is the closest number to both 12000 and 13000).

And so on.